### DISCUSSION TITLE
BlueRetro doesn't seem to work in Wii Remote EXT

### DISCUSSION THREAD

**Discussion by vaguerant** [darthcloud/BlueRetro#782] [Q&A] [answered]
I recently bought one of those AliExpress BlueRetro cores with hot-swappable cable setups. I purchased two cables with the core, a 2-port GameCube adapter, and a 2-port Wii EXT adapter. The GameCube adapter works perfectly, but the Wii EXT adapter does not.

I'm aware that using BlueRetro adapters in a Wii Remote is not really advised and that the intended purpose is the NES/SNES Classic Mini units, but since I see Wii Remote encryption was added in 1.8.2, I thought I might have hope. Unfortunately, I don't own a NES/SNES Classic Mini in order to determine whether the cables would function there.

I have the Universal firmware installed onto my core since I need to be able to hot-swap between GameCube and Wii. **EDIT:** I'm on `v1.8.4 hw1 universal`. **/EDIT**

I recall reading that HW1 devices need all their plugs inserted to function, so I plugged the adapter cable and core into two Wii Remotes. Powering up both remotes, the LEDs on the core remained completely unlit. However, if I press the EN button on the core, both LEDs light up while I have the button pressed and dim again as soon as I release the button.

Does anybody want to guess at what's going wrong here? The LEDs not being lit makes me wonder if this is a power draw issue, perhaps my Wiimotes aren't providing enough power to run the BlueRetro core? For now, it seems it just doesn't work plugged into two Wii Remotes. If there's any other diagnostic info I can provide, let me know.

**Comment by vaguerant** [comment, marked as answer]
> Does anybody want to guess at what's going wrong here? The LEDs not being lit makes me wonder if this is a power draw issue, perhaps my Wiimotes aren't providing enough power to run the BlueRetro core?

This seems to be the issue. By powering the BlueRetro over USB, I'm able to pair and use controllers by plugging into Wii EXT.

**Reply by darthcloud** [reply]
Yes Wiimote/Classic consoles do not provide enough power, so USB power is required.

It's mentioned in the cable docs https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#diy-through-hole-18
